src/engine/strat_engine/blockdev.rs

```diff
@@ -225,7 +241,7 @@ impl BlockDev {
         }
     }
 
-    pub fn wipe_metadata(&mut self) -> EngineResult<()> {
+    pub fn wipe_metadata(&self) -> EngineResult<()> {
```

Comment:
Opened issue https://github.com/stratis-storage/stratisd/issues/282 to track the issue.